docs/rhel-custom-image.md

```diff
@@ -0,0 +1,57 @@
+# RedHat Enterprise Linux
+
+Cloud providers which are listed below, support using rhel as an operating system option: 
+
+- AWS 
+- Azure
+- GCE
+- KubeVirt
+- Openstack
+- vSphere
+
+####  AWS:
+For amazon web service cloud provider, one the user applies for Gold Accesses Images using their RedHat Customer Portal, their request 
+will be reviewed and validated. Afterwards, new images will be added to the aws account under EC2-> Images-> AMIs-> Private Images.
+Once the images are available in the aws account, the image id for rhel(at the moment we only support 8.x.x versions) should be then 
+added to the `MachineDeployment` spec to the field `ami`.
+
+####  Azure
+Same for aws, RedHat supports Azure and provides images for it. Once those images are available, the field `imageID` in the 
+`MachineDeployment` spec should be updated with this image. 
+
+**Note:** 
+Azure rhel images starting from 7.6.x don't support cloud-init as their documentation states [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init#rhel).
+Thus, custom images can be used with a cloud-init pre-installed to solve this issue. Follow this [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cloudinit-prepare-custom-image)
+to prepare an image with cloud-init support.
+ 
+####  GCE
+RedHat also provides Gold Access Image for GCE and those can be fetched just like aws and azure. The image id should be added to 
```

Comment:
Similarly to my previous comments here I would link the RH documentation: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_red_hat_enterprise_linux_8_on_public_cloud_platforms/assembly_deploying-a-rhel-image-as-a-compute-engine-instance-on-google-cloud-platform_deploying-a-virtual-machine-on-aws